### PR TITLE
[W5][W16-3]Jitesh Khiani

### PR DESCRIPTION
**Pull request by jkhiani**
Unable to update tests due to timing related feature

### CODE REVIEW

**Comment by roanurquhart** [comment]
The amount of time a user spends using the application and the amount of commands he enters is useful information for not only the user, but also the developers because it allows us to tailor our work to our user's preferences. Good job.

**Comment by ZhangJiayu0303** [comment]
Nice! To count the number of commands is very cool! You may also consider to count repeat times for each command.

File: test/expected.txt [hunk 1]
```diff
@@ -303,3 +303,4 @@
 || Good bye!
 || ===================================================
 || ===================================================
+
```
Comment (lestertj) [inline test/expected.txt:306]: Does this test fully cover your feature? If it doesn't, it's a good idea to add some unit tests to cover it

**Review by lestertj — COMMENTED** [review]
Interesting feature, and well implemented! However, do not that you ideally have some form of tests to make sure your program is printing the right thing!

**Pull request closed without merging**